spec/models/custom_attribute_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe CustomAttribute do
+  it "return correct string field type" do
+    string_custom_attribute = FactoryGirl.build(:custom_attribute, :name => "foo", :value => "string", :resource_type => 'ExtManagementSystem')
```

Comment:
@lpichler from betterspecs:

> When you have to assign a variable instead of using a before block to create an instance variable,

That wasn't the case here - it was just a local variable in the example body which I think is a poor candidate for extracting to `let`, especially if it's an actor in the execution/verification steps. Readability (which can be subjective) IMO went :arrow_down: . But it's a small point that I think doesn't need to be changed, as you said :smile: